pytest.ini

```diff
@@ -3,15 +3,15 @@ python_files = *.py
 norecursedirs = .* _* node node_modules coverage venv
 addopts =
     -m "not integration"
-    --assert=plain
     --tb short
     --cov
-    --cov-report=term-missing
+    --cov-report=term:skip-covered
     --ds=config.settings.test
 
 filterwarnings =
-    ignore::DeprecationWarning
-    ignore::PendingDeprecationWarning
+    ignore:.*invalid escape sequence.*:DeprecationWarning
+    ignore:.*enqueued_at.*naive datetime.*:RuntimeWarning
+    ignore:.*success_at.*naive datetime.*:RuntimeWarning
```

Comment:
IMO ignoring all DeprecationWarnings is too aggressive and might cause us to miss important stuff. Indeed the [latest test run shows some important warnings](https://app.circleci.com/pipelines/github/oddbird/MetaDeploy/2376/workflows/33a5e4fe-a55b-4c99-b8e5-056fea201df5/jobs/13372) related to Django and pytest. I'll work on fixing those next (could be on this PR or another)

Reply:
I fixed most warnings in 5a6c727c797a67d27bf3c23a2936ba55b02c1561. I couldn't determine where this is coming from:

```
../usr/local/lib/python3.9/site-packages/selenium/webdriver/firefox/firefox_profile.py:208
  /usr/local/lib/python3.9/site-packages/selenium/webdriver/firefox/firefox_profile.py:208: SyntaxWarning: "is" with a literal. Did you mean "=="?
    if setting is None or setting is '':
```

And this one I don't really understand:

```
metadeploy/tests/rq_worker.py::TestConnectionClosingWorker::test_work
  /usr/local/lib/python3.9/site-packages/django/db/backends/postgresql/base.py:304: RuntimeWarning: Normally Django will use a connection to the 'postgres' database to avoid running initialization queries against the production database when it's not needed (for example, when running tests). Django was unable to create a connection to the 'postgres' database and will use the first PostgreSQL database instead.
    warnings.warn(

metadeploy/tests/rq_worker.py::TestConnectionClosingWorker::test_work
  /app:0: PytestWarning: Error when trying to teardown test databases: RuntimeError("generator didn't stop after throw()")
```

I'll add a card in case we want to investigate, but I think we can merge this PR now